schema.sql

```diff
@@ -0,0 +1,11 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+    id              INT GENERATED ALWAYS AS IDENTITY,
+    name            VARCHAR(250),
+    date_of_birth   DATE,
+    escape_attempts INT,
+    neutered        BOOLEAN,
+    weight_kg       DECIMAL,
+    PRIMARY KEY(id)
+);
```

Comment:
- [ ] **Good job so far 👏 👏** However, it was suggested in the project requirements to add a screenshot of all query results to the pull request description. 😂 Kindly, add them as the result of all your queries 👍
 
![image](https://user-images.githubusercontent.com/102757126/228016959-bf07ba38-a9b5-4547-9e48-4390f1638a83.png)

Reply:
Hello @TuyishimireEric ,
Thanks for your review